releasenotes/notes/update-number-to-2q-clifford-mapping-c28f1f29b0205d57.yaml

```diff
@@ -0,0 +1,20 @@
+---
+fixes:
+  - |
+    Fix a bug that :meth:`.ClifforUtils.random_cliffords` always returned a single Clifford
```

Comment:
On this line and another one:
"Fixed a bug where :meth:`.CliffordUtils.random_cliffords`"
(most notably, the missing "d")

Reply:
Thank you for catching typos and grammar errors in docs. I've fixed all of them in [aff3fb5](https://github.com/Qiskit/qiskit-experiments/pull/940/commits/aff3fb575afeb8c65d77a6521025c18365f21e4d).